fittable.R

```diff
@@ -0,0 +1,36 @@
+fittable <- function(..., indices="robust", all=F) {
+  require(lavaan)
+  models <- list(...)
+  indic.names <- c("x2", "df", "p", "BIC", "CFI", "TLI", "RMSEA", "lower.RMSEA", "upper.RMSEA", "SRMR")
+  
+  if (indices == "scaled") {
+    indic <- c("chisq.scaled", "df.scaled", "pvalue.scaled", "bic", "cfi.scaled", "tli.scaled", "rmsea.scaled",
+               "rmsea.ci.lower.scaled", "rmsea.ci.upper.scaled", "srmr")
+
+  } else if (indices == "robust") {
+    indic <- c("chisq.scaled", "df.scaled", "pvalue.scaled", "bic", "cfi.robust", "tli.robust", "rmsea.robust",
+               "rmsea.ci.lower.robust", "rmsea.ci.upper.robust", "srmr")
+  } else {
+    indic.names <- indic <- indices

```

Comment:
Jestli tomu rozumim spravne, tak tohle je tu pro to, ze nekdy muzes chtit jako `indices` passovat vektor jinych nez tech defaultne ulozenych indexu co mas na radcich 7-12?

Reply:
Jo. Původně jsem tam chtěl mít `...`, ale nevěděl jsem, jak to pak načíst :-)